doc/doxygen/src/advanced-build-system-tricks.md

```diff
@@ -164,13 +164,68 @@ So the current filename should be higher than 60-serial.rules
 If for some reason re-writing the serial is needed there is a windows tool:
   https://remoteqth.com/wiki/index.php?page=How+to+set+usb+device+SerialNumber
 
+
 Documentation:
 --------------
 * The whole documentation
   http://reactivated.net/writing_udev_rules.html#udevinfo
 * Udev manpage
   http://manpages.ubuntu.com/manpages/eoan/en/man7/udev.7.html
 
+# Handling multiple boards without udev-rules
+=============================================
+
+This is a simpler approach to the above mentioned issue. The solution here only
+uses a makefile script for selecting the debugger and serial port. No
+adminstrative privileges (e.g. to configure Udev) are required.
+
+One of the limitations of the solution described here is that it currently
+doesn't work with multiple boards of the same type.
```

Comment:
Are you sure of this ? Isn't the serial number supposed to be used by the build system to flash and open a serial on the right board ? I thought OpenOCD, edbg and a few other flashing tool were able to do that.

Reply:
The limitation here is because the script doesn't support multiple boards of the same type at the same time. It could be solved by adding an optional index to the mapping:
```
SERIAL_samr21-xpro_1 ?= ######
SERIAL_samr21-xpro_2 ?= ######
```

And match a BOARD_NUM variable to that. But I consider that out of scope here :)

Reply:
Ah yes, that makes indeed :)
But, then maybe extend the comment to explain the limitation.